tasks/proftpd.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Install proftpd
+  apt:
+    pkg: proftpd
+    state: latest
+
+- name: Start proftpd Service
+  service:
+    name: proftpd
+    state: started
+    enabled: yes
```

Comment:
On Debian-like systems (Ubuntu here) the installed service is typically started and enabled automatically, so that second task isn't necessary.